CONTRIBUTING.md

```diff
@@ -206,24 +198,28 @@ The following checklist is used when reviewing pull requests
 ### General (all submissions)
 
 - Commit messages are useful and properly formatted
+- No sensitive personally identifiable information (PII) is included
 - Unit tests validate the code submission
 - One commit, or at most only a handful.  More than five commits should
   be heavily questioned
 
+To find out more about sensitive PII, refer to [Sensitive PII versus non-sensitive PII](https://www.ibm.com/topics/pii#Sensitive+PII+versus+non-sensitive+PII)
+
 
-### Parsers
+### Component (Datasources, Components, Parsers, and Combiners)
 
-- Parser is properly documented and should include:
+- Component is properly documented and should include:
    - Example input
-   - The resulting data structure represented by the parser
-   - Parser usage is clear to a user with some knowledge of the domain
+   - The resulting data structure represented by the component
+   - Component usage is clear to a user with some knowledge of the domain
      without needing to examine the code itself
    - Meaning and usage of an "empty" (falsy data object) is clear
 
 - Unit tests cover both positive and negative cases and utilizes
   reasonable examples of input data. Test data should be usable in the
   generation in archives used for integration testing and product
   demonstrations.
+  - 100 branch coverage is recommended
```

Comment:
@xiangce It seems you mean to type `100% branch coverage` here. 

Also, it would be better if we can adjust the indents to the same for this subitem and the subitems above somehow.